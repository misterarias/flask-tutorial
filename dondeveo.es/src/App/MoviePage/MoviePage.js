import React from 'react';
import { Row } from 'reactstrap';
import './MoviePage.css' ;

const MoviePage = (props) => {
  return (
    <Row>
      <h1>Soy una muvi</h1>
    </Row>
  );
}

export default MoviePage;
