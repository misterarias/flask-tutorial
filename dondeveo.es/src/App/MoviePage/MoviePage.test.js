'use strict'

import React from 'react';
import MoviePage from './MoviePage';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <MoviePage />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});

