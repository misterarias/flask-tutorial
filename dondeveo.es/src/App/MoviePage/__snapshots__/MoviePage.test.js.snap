// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="row"
>
  <h1>
    Soy una muvi
  </h1>
</div>
`;
